libs/config/config.py

```
@@ -1,18 +1,113 @@
+import json
+import logging
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)